app/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= render partial: "breadcrumbs" %>
 
 
-  <nav aria-label="Menu" class="profile-menu">
+  <nav aria-label="Menu" class="profile-menu" data-person-email="<%= current_person.email %>">
```

Comment:
Is `current_person` guaranteed to be non-nil? What is this set to if the person is not logged in?

Reply:
It returns an instance of the `Guest` class, which has a nil email.